### PR TITLE
Add fastlane support

### PR DESCRIPTION
**Pull request by AlexYanski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +123,21 @@ For formating run the sctipt:
 sh ./Scripts/swiftformat.sh
 ```
 
+**Fastlane**
+
+To use **Fastlane** you need to run the script. It will create a `Fastfile` configuration file in which you will add a building options. Those options you can find at the [documentation](https://docs.fastlane.tools).
```
Comment (pkondrashkov) [inline README.md]: This is redundant, we have already set up Fastfile in the project.

File: README.md [hunk 2]
```diff
@@ -111,6 +123,21 @@ For formating run the sctipt:
...
+
+```bash
+bundle exec fastlane init
+```
+
+To use a your configuration that you added in `Fastfile`, run the scrip:
```
Comment (pkondrashkov) [inline README.md]: That is good example, but would be nice to have a link to readme generated by fastlane here. "For more information check out fastlane [README.md]. Something like that

File: README.md (hunk 1 above)
Comment (AlexYanski) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (AlexYanski) [reply README.md]: Done

**Pull request merged**